tests/rptest/tests/upgrade_test.py

```diff
@@ -614,3 +614,37 @@ def cloud_log_size() -> int:
         # Assert manifest has not been mutated by comparing sizes
         total_cloud_size_after_upgrade = cloud_log_size()
         assert total_cloud_size_before_upgrade <= total_cloud_size_after_upgrade, f"Mismatch in cloud storage size after upgrade, bytes before: {total_cloud_size_before_upgrade} bytes_after: {total_cloud_size_after_upgrade}"
+
+
+class InstallSelfTest(RedpandaTest):
+    @cluster(num_nodes=1)
+    def test_redpandainstaller_install(self):
+        """
+        Smoke test that checks RedpandaInstaller.install
+        :return:
+        """
+        install = lambda version: self.redpanda._installer.install(
+            self.redpanda.nodes, version)
+        start = lambda: self.redpanda.start(self.redpanda.nodes[0:1])
+        get_version = lambda: self.redpanda.get_version(self.redpanda.nodes[0])
```

Comment:
nit: these one-liners aren't too verbose, and are fairly self-describing. Consider in-lining rather than using lambdas (easier to read quickly IMO since most folks are/will be familiar with the test primitives already)